libs/ob-sql-parser/src/main/resources/oboracle/sql/OBParser.g4

```diff
@@ -4575,7 +4752,7 @@ json_exists_response_type
     ;
 
 json_query_expr
-    : JSON_QUERY LeftParen js_doc_expr Comma js_literal (RETURNING js_query_return_type)? TRUNCATE? scalars_opt? PRETTY? ASCII? wrapper_opts? json_query_on_opt? RightParen
+    : JSON_QUERY LeftParen js_doc_expr Comma js_literal (RETURNING js_query_return_type)? TRUNCATE? scalars_opt? PRETTY? ASCII? wrapper_opts? json_query_on_opt? MULTIVALUE? RightParen
```

Comment:
`... wrapper_opts? ASIS? json_query_on_opt?...`, there should be a token 'ASIS'

Reply:
Yes you are right. I updated it.